REvernus/Settings/HotkeySettings.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace REvernus.Settings
+{
+    [Serializable]
+    public class HotkeySettings
+    {
+        public string MarketUpHotkey { get; set; } = "Control+X";
```

Comment:
IMO Control+X and Control+Z are not great default keybinds, mainly because they are common shortcuts for undo and cut, maybe something like Control-Q/W may be better.